javaScript-week2/flightBooking.js

```diff
@@ -0,0 +1,21 @@
+function getFullname(firstname, surname, useFormalName, isFemale) {
+  if (firstname === "" && surname === "") {
+    return alert("Please enter your name");
+  } else if (useFormalName) {
+    if (isFemale) {
+      return `Madam ${firstname} ${surname}`;
```

Comment:
You've repeated `${firstname} ${surname}` 3 times. A good practice is to create a variable and reuse it instead. This is a [coding principle called DRY (Don't Repeat Yourself)](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).
```
const fullName = `${firstname} ${surname}`;
```